### PR TITLE
[Merged by Bors] - chore: address some porting notes for Jacobson ideals

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

- [x] depends on: #19734

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — APPROVED** [review]
Thanks! Your commit alone looks good to me.
The dependency still needs to land first: dear maintainers, please **don't merge just yet** :-) .
maintainer delegate?

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**
